Sources/Plasma/CoreLib/plFileSystem.cpp

```diff
@@ -479,19 +483,21 @@ plFileName plFileSystem::GetUserDataPath()
 #if defined(HAVE_BUILTIN_AVAILABLE) && defined(HAVE_SYSDIR)
         if (__builtin_available(macOS 10.12, *)) {
             sysdir_search_path_enumeration_state state;
-            state = sysdir_start_search_path_enumeration(SYSDIR_DIRECTORY_APPLICATION_SUPPORT, SYSDIR_DOMAIN_MASK_USER);
+            state = sysdir_start_search_path_enumeration(SYSDIR_DIRECTORY_DOCUMENT, SYSDIR_DOMAIN_MASK_USER);
```

Comment:
This might not be right for macOS?

Reply:
I did this for diagnostic reasons (takes a while to pull down App Support.) I'll undo.